src/datetime-score-api/app.py

```diff
@@ -0,0 +1,36 @@
+import numpy as np
+import pandas as pd
+from flask import Flask, abort, jsonify, request, url_for
+import random
+import sqlalchemy as db
+from functools import reduce
+
+
+app = Flask(__name__)
+
+file = []
+with open('tickers.txt','r') as inf:
+    for line in inf:
+        file.append(eval(line))
+ticker_aliases = file[0]
+
+
+@app.route('/', methods=['POST'])
+def data_search(search):
+   engine = db.create_engine('postgresql+psycopg2://HIDDENALPHABET:password@hiddenalphabet-db.ceqxyonbodui.us-east-1.rds.amazonaws.com:5432/HiddenAlphabet')
+   connection = engine.connect()
+   metadata = db.MetaData()
+   testtable = db.Table('reddit-comments', metadata, autoload=True, autoload_with=engine)
+   query = db.select([testtable])
+   ResultProxy = connection.execute(query)
+   ResultSet = ResultProxy.fetchall()
+   CORPUS = pd.DataFrame(ResultSet)
```

Comment:
This is no longer in global scope, and not a constant in this context
```suggestion
   corpus = pd.DataFrame(ResultSet)
```

Reply:
Fixed.